storage/innobase/xtrabackup/doc/source/advanced/page_tracking.rst

```diff
@@ -0,0 +1,97 @@
+.. _page_tracking:
+
+====================================
+Page Tracking for Incremental Backup 
+====================================
+
+This feature is *tech preview* quality.
+
+Percona XtraBackup 8.0.27 adds support for the page tracking functionality for `incremental backup <https://www.percona.com/doc/percona-xtrabackup/8.0/backup_scenarios/incremental_backup.html>`__ . 
+
+To create an incremental backup using page tracking, the *mysqlbackup* looks for pages that have been modified since the last backup in the InnoDB data files and copies only changed pages. 
```

Comment:
Can you add something like this 
PXB, with the page tracking feature, uses the mysqlbackup component of MySQL to retrieve the modified pages for the incremental backup.

Reply:
Updated